server/controllers/payment.js

```diff
@@ -0,0 +1,37 @@
+const User = require("../models/User");
+const stripe = require("stripe")(process.env.STRIPE_SECRET_KEY);
+
+const asyncHandler = require("express-async-handler");
+
+exports.createSubcription = asyncHandler(async (req, res, next) => {
+  const customer = await stripe.customers.create({
+    email: "timbits.rosen@example.com",
```

Comment:
Couldn't we make this the authenticated users email as opposed to static email, if this is just an initial implementation then this PR should be in draft and not ready to merge?

Reply:
I will remove controlle from this pr. I just created it for testing purposes and left it there in case someone else takes the integration of stripe payment so they would have a general idea about how things will be implemented.